crates/solidity/inputs/schema/grammar/03-types/04-elementary-types/productions.yml

```diff
@@ -27,6 +41,17 @@
   unversioned:
     reference: "PayableKeyword"
 
+- name: "ByteType"
+  kind: "Scanner"
```

Comment:
similarly to how we defined everything else like `AddressType` and `PayableType`, I suggest having a `ByteType` parser that references a `ByteKeyword` scanner (in the keywords spec topic), since it is an actual keyword, and should be included anywhere else we define/use keywords.

Reply:
But it is parallel to FixedBytesType, for which it is an alias